schemas/dto/entityProcessingSummary.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Describes entity processing summary of the certain entity type",
+  "javaType": "org.folio.rest.jaxrs.model.EntityProcessingSummary",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "totalCreatedEntities": {
+      "description": "Total number of created entities",
+      "type": "integer"
+    },
+    "totalUpdatedEntities": {
+      "description": "Total number of updated entities",
+      "type": "integer"
+    },
+    "totalDiscardedEntities": {
+      "description": "Total number of discarded entities, consists of total errors and number of occurrences that entity does not match",
+      "type": "integer"
+    },
+    "totalErrors": {
+      "description": "Total errors number during entity processing",
+      "type": "integer"
+    }
+  },
+  "required": [
+    "jobExecutionId"
```

Comment:
Hi @RuslanLavrov there is no jobExecutionId property, could you please add it. Thank you

Reply:
Hi @AivarIUsupov, in fact, "jobExecutionId" property is not needed for this schema, I will delete this mention from here. Thank you!